dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -391,8 +391,9 @@ export const Card = ({
 				<SupportingContent
 					supportingContent={supportingContent}
 					containerPalette={containerPalette}
-					alignment={supportingContentAlignment}
-					isDynamo={isDynamo}
```

Comment:
`isDynamo` was only being used to fix alignment to 'horizontal' so ive moved this prop out and let the card decide it in the alignment prop